resources/import.sql

```diff
@@ -1,6 +1,6 @@
 -- comment this out if your server has a different table/location for phone_number
 -- or if it already exists in the users table
-ALTER TABLE `users` ADD COLUMN `phone_number` VARCHAR(20) NULL DEFAULT NULL;
+ALTER TABLE `users` ADD COLUMN `phone_number` VARCHAR(20) DEFAULT NULL;
```

Comment:
Can we add something like `IF NOT EXISTS` as we do in the table generation so that people don't have to edit this file if they already have this column?

Reply:
So as far as I understand from researching this, different mysql versions handle this whole `IF NOT EXIST` differently so there wouldn't be a universal case for this. We can always include a way to drop the SQL i guess:
`ALTER TABLE `users` DROP COLUMN `phone_number`;`. 

I agree there needs to be a better way to do this since in the past with gcphone just the one column not being proper caused issues if they came from another phone.